src/plugins/intel_cpu/src/cpu_memory.h

```diff
@@ -67,6 +67,8 @@ class IMemoryMngr {
      * @return status whether the object has control over underlying memory buffer
      */
     virtual bool hasExtBuffer() const noexcept = 0;
+
+    virtual size_t getSize() const noexcept = 0;
```

Comment:
This call is not necessary. The proxy may keep the last requested size internally and call resize every time the new memory manager is set, to insure that there is enough memory allocated.

Reply:
done